Fix power extraction when plotlist is not empty

sporrasm opened the pull request:
If plotlist contains elements, the manually probed signals are saved in print file before supplies and after the actual signals that are attached to any IOs. In this case, the output file would be read incorrectly, as only labels matching to spice_iofile.ionames would be read.

mkosunen commented:
@kspoof , Could you try to run sum analog sims with the latest commit with Ngspice? @sporrasm has tested AC and tran for spectre.

Also close #13 #14 #15 and #16 if they have been implemented already.

kspoof commented:
@mkosunen ngspice transient and AC are now OK

Commit:
Fix power extraction when plotlist is not empty

If plotlist contains elements, the manually probed signals are saved in
print file before supplies and after the actual signals that are
attached to any IOs. In this case, the output file would be read
incorrectly, as only labels matching to spice_iofile.ionames would be
read.

vlahtin commented:
Moi,
nothing seems to break for my test cases at least.
